deploy/vault-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keyring-secret-vault
+type: Opaque
+stringData:
+  keyring_vault_n1.conf: |-
```

Comment:
```suggestion
  keyring_vault.conf: |-
```

Reply:
Done as a part of https://github.com/percona/percona-xtradb-cluster-operator/pull/349/commits/7f1743fe00046d3b32e2808ab7b11032f1d3ea62